terraform/helm/aptos-node/templates/configmap.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "aptos-validator.fullname" . }}
+  labels:
+    {{- include "aptos-validator.labels" . | nindent 4 }}
+data:
+{{ tpl (.Files.Glob "files/configs/*.yaml").AsConfig . | indent 2 }}
+
+{{- if .Values.loadTestGenesis }}
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "aptos-validator.fullname" . }}-genesis-e{{ .Values.chain.era }}
+  labels:
+    {{- include "aptos-validator.labels" . | nindent 4 }}
+data:
+  waypoint.txt: |-
+{{ .Files.Get "files/test-data/waypoint.txt" | indent 4 }}
+  secure-data.json: |-
```

Comment:
should we store `secure-data.json` in a k8s secret instead?

Reply:
Ideally I want the tool to create this one automatically, for now just a placeholder to show what the file content looks like